.gitignore

```diff
@@ -8,6 +8,7 @@
 /.idea/
 *.iml
 nbactions.xml
+nb-configuration.xml
```

Comment:
Unrelated, but ok.

Reply:
oups my bad,  I'll fix my git ignore.

Reply:
I wrote "unrelated" because it's unrelated to the change you're making in this PR. Usually it is good practice to split this kind of changes into different PRs as it makes them easier to review and to audit later in case of regressions.

Tip: It's possible to configure a global `.gitignore` for your user https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Reply:
I think I managed to resolve all the issues. can we leave the .gitignore to be included in this pr?
I really think having the NetBeans config in the git ignore is harmless and might help others.
having them in the global config file will make my life more complicated.